inc/functions.inc

```diff
@@ -9620,20 +9620,27 @@ if (!function_exists("livesearch_tuotehaku")) {
         // Katsotaan mist� normaalivarastoista k�ytt�j� saa myyd�
         if (in_array($yhtiorow['livetuotehaku_hakutapa'], array('E', 'G', 'I', 'K', 'M'))) {
 
-          $varastolisa = $kukarow['varasto'] != '' ? "AND tunnus IN ({$kukarow['varasto']})" : "";
+          if (!empty($laskurow['varasto'])) {
+            $laskurow['varasto'] = (int) $laskurow['varasto'];
+            $varastolisa = " AND tunnus IN ({$laskurow["varasto"]})";
```

Comment:
Miten tässä voisi olla usea tunnus (`tunnus IN`), kun ylempänä sanotaan `(int) $laskurow['varasto']`?

Lisäksi tässä muutetaan laskurown sisältöä, joka on globaali. Varmaan tulisi olla esim;

```php
$laskun_varasto = (int) $laskurow['varasto'];
$varastolisa = " AND tunnus = {$laskun_varasto} ";